src/App.css

```diff
@@ -27,8 +27,18 @@
 .addColorInputBtnContainer {
   display: flex;
   margin: 0.5rem 0 2rem;
+  column-gap: 0.5rem;
+}
+.addColorInputBtnContainer .addColorBtn {
+    background-color: var(--text);
+}
+.addColorInputBtnContainer .resetBtn {
```

Comment:
Can you make the text and the icon on the reset button dark?

Reply:
on it